README.md

```diff
@@ -12,6 +12,13 @@ Usage instructions
 3. `sudo apt-get update`
 4. `sudo apt-get install discord`
 
+Commands:
+
+```shell
+curl -fSsL https://palfrey.github.io/discord-apt/discord-apt.gpg.asc | gpg --dearmor | sudo tee /etc/apt/keyrings/discord-apt.gpg > /dev/null
```

Comment:
This is a different command to the one I listed - different folder for starters and the `dearmor` bit

Reply:
so the dearmor is required for apt to interpret it

the new location is there so that these keys do not get used for other apt repos, its essentially like scoping them down.

since the update and install commands are functional, i did not feel the need to repeat those.

i did not update the steps you have described because i did not feel the need to, they serve the purpose of being understandable to someone reading for the first time. i can edit if you prefer.